cargo.nut

```diff
@@ -443,10 +466,75 @@ function DefineCargosBySettings(economy)
 		case(Economies.ECS): //ECS 1.2
 			::CargoLimiter <- [0,2];
 			::CargoCat <- [[0,2,31],
-				       [4,6,13,14,22,27],
-				       [1,3,7,8,10,15,17,26,29],
-				       [9,12,18,19,20,21,23],
-				       [5,11,24,25,28]];
+						  [4,6,13,14,22,27],
+						  [1,3,7,8,10,15,17,26,29],
+						  [9,12,18,19,20,21,23],
+						  [5,11,24,25,28]];
+			::CargoCatList <- [CatLabels.PUBLIC_SERVICES,CatLabels.RAW_FOOD,CatLabels.RAW_MATERIALS,
+					   CatLabels.PROCESSED_MATERIALS,CatLabels.FINAL_PRODUCTS];
+			::CargoMinPopDemand <- [0,500,1000,4000,8000];
+			::CargoPermille <- [60,25,25,15,10];
+			::CargoDecay <- [0.4,0.2,0.2,0.1,0.1];
+			break;
+		case(Economies.FIRS2__TEMPERATE_BASIC): // FIRS 2 - Temperate Basic
+			::CargoLimiter <- [0,2];
+			::CargoCat <- [[0,2],
+						  [9,10,13,16],
+						  [4,6,12,17,18],
+						  [3,7,8,14,15],
+						  [1,5,11]];
+			::CargoCatList <- [CatLabels.PUBLIC_SERVICES,CatLabels.RAW_FOOD,CatLabels.RAW_MATERIALS,
+					   CatLabels.PROCESSED_MATERIALS,CatLabels.FINAL_PRODUCTS];
+			::CargoMinPopDemand <- [0,500,1000,4000,8000];
+			::CargoPermille <- [60,25,25,15,10];
+			::CargoDecay <- [0.4,0.2,0.2,0.1,0.1];
+			break;
+			case(Economies.FIRS2__ARCTIC_BASIC): // FIRS 2 - Arctic Basic
+			::CargoLimiter <- [0,2];
+			::CargoCat <- [[0,2],
+						  [11],
+						  [1,4,9,13,14,15,16,19],
+						  [3,6,7,8,10,12,17],
+						  [5,18]];
+			::CargoCatList <- [CatLabels.PUBLIC_SERVICES,CatLabels.RAW_FOOD,CatLabels.RAW_MATERIALS,
+					   CatLabels.PROCESSED_MATERIALS,CatLabels.FINAL_PRODUCTS];
+			::CargoMinPopDemand <- [0,500,1000,4000,8000];
+			::CargoPermille <- [60,25,25,15,10];
+			::CargoDecay <- [0.4,0.2,0.2,0.1,0.1];
+			break;
+			case(Economies.FIRS2__TROPIC_BASIC): // FIRS 2 - Tropic Basic
+			::CargoLimiter <- [0,2];
+			::CargoCat <- [[0,2],
+						  [3,6,12,13,14,15],
+						  [8,17,18,19],
+						  [4,9,10,16],
+						  [1,5,7,11]];
```

Comment:
I would say Alcohol is in the Raw food category, because it is directly produced by vineyard, which is a raw industry.

```suggestion
						  [1,3,6,12,13,14,15],
						  [8,17,18,19],
						  [4,9,10,16],
						  [5,7,11]];
```